app/assets/styles/base/banner.scss

```diff
@@ -1,112 +1,144 @@
-                height: 100%;
-                display: flex;
-                align-items: center;
+            > .btn {
+                padding: 2px 10px;
+
+                .fa {
+                    margin-right: 5px;
+                }
+
             }
 
-            > [message] {
-                padding: 5px 24px;
-                flex: 1;
+            > .other-fixes-btn {
+                border-left: none;
+                border-top-left-radius: 0;
+                border-bottom-left-radius: 0;
+                padding: 5px 5px 5px 8px;
+                margin-left: -3px;
             }
+        }
 
-            > .quick-fix-btn-container {
-                padding: 0 10px;
-                position: relative;
-                border-right: 1px solid $red-dark;
-                border-left: 1px solid $red-dark;
-                height: 100%;
-                display: flex;
-                align-items: center;
-
-                > .btn, > .other-fixes-btn {
-                    border-radius: 3px;
-                    cursor: pointer;
-                    height: 28px;
-                    &:hover:not(.disabled) {
-                        background: $red-dark;
-                    }
-
-                    &.disabled {
-                        cursor: default;
-                    }
-                }
+        > .caret {
+            width: 20px;
+            font-size: 16px;
+        }
+    }
 
-                > .btn {
-                    border: 3px solid $red-dark;
-                    padding: 2px 10px;
+    > .details-container {
+        padding: 10px 24px;
+        border-top: 1px solid $red-dark;
+        font-weight: 500;
+    }
+}
 
-                    .fa {
-                        margin-right: 5px;
-                    }
 
-                }
+@mixin banner-scheme($main, $accent) {
+    background: $main;
 
-                > .other-fixes-btn {
-                    border: 3px solid $red-dark;
-                    border-left: none;
-                    border-top-left-radius: 0;
-                    border-bottom-left-radius: 0;
-                    padding: 5px 5px 5px 8px;
-                    margin-left: -3px;
-                }
-            }
+    > .summary-container {
+        > [code] {
+            border-right: 1px solid $accent;
+        }
 
-            > .caret {
-                width: 20px;
-                font-size: 16px;
+        > .quick-fix-btn-container {
+            border-right: 1px solid $accent;
+            border-left: 1px solid $accent;
+
+            > .btn, > .other-fixes-btn {
+                border: 3px solid $accent;
+
+                &:hover:not(.disabled) {
+                    background: $accent;
+                }
             }
         }
 
         > .details-container {
-            padding: 10px 24px;
-            border-top: 1px solid $red-dark;
-            font-weight: 500;
+            border-top: 1px solid $accent;
         }
     }
+
+    .other-fixes {
```

Comment:
In the banner there is the primary fix which is displayed directlry and then other fixes shown only after over